src/Grpc.Net.Client/Internal/NtDll.cs

```diff
@@ -0,0 +1,69 @@
+#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+#if !NET5_0_OR_GREATER
+
+using System.Runtime.InteropServices;
+
+namespace Grpc.Net.Client.Internal;
+
+internal static class NtDll
```

Comment:
Looks like this code is copied from somewhere, can you pls point to the original location for reference? (or to a place where this approach is at least partially documented).

Reply:
https://www.pinvoke.net/default.aspx/ntdll/RtlGetVersion.html was linked where this type is used. I've added it to this file.